Document/0x05b-Basic-Security_Testing.md

```diff
@@ -279,102 +279,19 @@ Frida also provides bindings for various languages, including Python, C, NodeJS,
 
 ##### Drozer
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1233>
+[Drozer](https://github.com/mwrlabs/drozer "Drozer on GitHub") is an Android security assessment framework that allows you to search for security vulnerabilities in apps and devices by assuming the role of a third-party app interacting with the other application's IPC endpoints and the underlying OS.
 
-[Drozer](https://github.com/mwrlabs/drozer "Drozer on GitHub") is an Android security assessment framework that allows you to search for security vulnerabilities in apps and devices by assuming the role of a third-party app interacting with the other application's IPC endpoints and the underlying OS. The following section documents the steps necessary to install and use Drozer.
+###### Drozer Setup
```

Comment:
As it's becoming now the 5th level in the title, let's remove it. It anyway has then a nice flow when reading the paragraphs

Reply:
done, titles removed